### PR TITLE
Add new StringTie 1.1.0 package

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: packages/package_stringtie_1_1/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+categories:
+- Tool Dependency Packages
+description: Contains a tool dependency definition that downloads and installs version
+  1.1 of StringTie.
```
Comment (nsoranzo) [inline packages/package_stringtie_1_1/.shed.yml]: s/1.1/1.1.0/

Comment (bgruening) [inline packages/package_stringtie_1_1/.shed.yml]: As you like. This binary was not renamed like this. So who knows how they will rename the next release. Imho the next release can be called 1_1_1 and this 1_1. But I don't mind changing the version.

File: packages/package_stringtie_1_1/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                </action>
+            </actions_group>
+        </install>
+        <readme>StringTie is a fast and highly efficient assembler of RNA-Seq
+        alignments into potential transcripts. It is primarily a genome-guided
+        transcriptome assembler, although it can borrow algorithmic techniques
+        from de novo genome assembly to help with transcript assembly. Its input
+        can include not only the spliced read alignments used by reference-based
+        assemblers, but also longer contigs that were assembled de novo from
+        unambiguous, non-branching parts of a transcript.</readme>
```
Comment (nsoranzo) [inline packages/package_stringtie_1_1/tool_dependencies.xml]: I think this description should match the one in `.shed.yml`.

File: packages/package_stringtie_1_1/.shed.yml (hunk 1 above)
Comment (nsoranzo) [inline packages/package_stringtie_1_1/.shed.yml]: The tgz are all named `1.1.0` and I also get:
```
$ stringtie
StringTie v1.1.0 usage:
...
```
So yes, I'd prefer 1.1.0

**Comment by davebx** [comment]
:+1: And I agree with @nsoranzo on using the full version string.

File: tools/stringtie/stringtie.xml [hunk 3]
```diff
@@ -55,14 +58,20 @@ stringtie "$input_bam"
                 <param help="-a" label="Minimum anchor length for junctions" name="min_anchor_len" type="integer" value="10" />
                 <param help="-j" label="Minimum junction coverage" name="min_anchor_cov" type="integer" value="1" />
                 <param help="-c" label="Minimum bundle reads per bp coverage to consider for assembly" name="min_bundle_cov" type="integer" value="2" />
-                <param help="-s" label="Coverage saturation threshold" name="maxcov" type="integer" value="1000000" />
                 <param help="-g" label="Gap between read mappings triggering a new bundle" name="bdist" type="integer" value="50" />
                 <param help="-M" label="Fraction of bundle allowed to be covered by multi-hit reads" name="bundle_fraction" type="float" value="0.95" />
+                <!--
+                This one requires sequence ID's? Not clear how we should use this.
+                param argument="-x" truevalue="-x" falsevalue="" type="boolean" help="" label="Do not assemble any transcripts on these reference sequence(s)" /-->
```
Comment (nsoranzo) [inline tools/stringtie/stringtie.xml]: By looking at the C++ source code it seems to me that this parameter should be followed by a comma(or space)-separated string.

Comment (bgruening) [inline tools/stringtie/stringtie.xml]: From my understanding it is a comma separated list of IDs. But this does not make much sense in this tool. At least not for me. Sequence ID's? From a BAM file?

Comment (nsoranzo) [inline tools/stringtie/stringtie.xml]: Maybe @gpertea can shed some light on this.

Comment (gpertea) [inline tools/stringtie/stringtie.xml]: The value of this parameter is a string which is a comma or space separated list of *reference* sequence IDs (i.e. chromosome/contig names) . E.g. ``-x chrM`` -- if one does not care at all about mitochondrial transcripts (and wants to save some processing time..); obviously these sequence IDs must precisely match the ones in the BAM header.

Comment (bgruening) [inline tools/stringtie/stringtie.xml]: @gpertea awesome!
Any change to make this work with a BED file? Discarding unwanted regions?

Comment (gpertea) [inline tools/stringtie/stringtie.xml]: That would be a natural extension of this option, indeed. Not currently on our high priority list of things to implement in StringTie, but I'll put it in as a  "feature requests"..

**Comment by bgruening** [comment]
@nsoranzo @davebx everything updated!

**Comment by nsoranzo** [comment]
@bgruening You should also move `packages/package_stringtie_1_1` directory to `packages/package_stringtie_1_1_0` .

**Comment by bgruening** [comment]
@nsoranzo done.

**Pull request merged**